powers.c

```diff
@@ -0,0 +1,27 @@
+/* Drew French */
+
```

Comment:
Close, but not quite. For every number input, one of the values that should be printed isn't. You also forgot 2^0 = 1 for each case.